support/installing-protoc.mdx

```diff
@@ -2,28 +2,57 @@
 title: "Installing Protoc"
 ---
 
-Since `v0.12.0` `protoc` is a requirement to use and install `cargo-shuttle`. To install `protoc`, here
-are some instructions for Windows, Linux and Mac:
+Since shuttle v0.12.0, `protoc` is a requirement to install `cargo-shuttle` and to use `cargo-runtime`.
+Here are some instructions for installing `protoc` on Windows, Linux and Mac:
 
 ### Windows
 
 On Windows the simplest way to install `protoc` is to first install [chocolatey](https://chocolatey.org/install), 
 and then simply run: `choco install protoc`.
 
+Then, verify that it's working:
+
+```bash
+# should show a version number
+protoc --version
+```
+
+**Note:** *You might need to restart your shell.
+If you are using the integrated terminal in VS Code, you need to restart VS Code.*
+
 ### Linux and Mac
 
-For Linux and Mac you can use this install script:
+For Linux and Mac you can use these commands:
 ```bash
-# replace <arch> with your architecture, for example for linux x86_64 replace <arch> with: linux-x86_64
-curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v21.9/protoc-21.9-<arch>.zip &&\
-    sudo unzip -o protoc-21.9-<arch>.zip -d /usr/local bin/protoc &&\
-    sudo unzip -o protoc-21.9-<arch>.zip -d /usr/local 'include/*' &&\
-    rm -f protoc-21.9-<arch>.zip
+# Replace <arch> with your architecture, for example:
+# Linux with Intel/AMD:  linux-x86_64
+# Mac with Intel:        osx-x86_64
+# Mac M1 etc:            osx-aarch_64
+# see https://github.com/protocolbuffers/protobuf/releases for all alternatives
+ARCH="<arch>" && \
+VERSION="22.2" && \
+curl -OL "https://github.com/protocolbuffers/protobuf/releases/download/v$VERSION/protoc-$VERSION-$ARCH.zip" && \
+sudo unzip -o "protoc-$VERSION-$ARCH.zip" bin/protoc "include/*" -d /usr/local && \
```

Comment:
I see you put this in "bin/protoc" rather than the previous "/usr/local/bin/protoc", I think that may be a problem on mac. We had to change it for our binary build to work on mac: https://github.com/shuttle-hq/shuttle/pull/731/commits/d09356d544dff238df8dd257d5cdc710cb94eb60

Reply:
The unzip command is functionally equivalent to your version, I just combined the two. The output dir is `-d /usr/local`.